tests/unit/igraph_density.c

```diff
@@ -27,7 +27,7 @@
 void test_density(const igraph_t *graph, igraph_bool_t loops) {
     igraph_real_t density;
 
-    if (igraph_density(graph, &density, loops)) {
+    if (igraph_density(graph, NULL, &density, loops)) {
```

Comment:
Should we add tests for the weighted calculation as well?

Reply:
Done.